test/uhmaeat.test.js

```diff
@@ -1,104 +0,0 @@
-// vim: tw=120 softtabstop=4 shiftwidth=4
```

Comment:
ah i see, because now we cant compare differences,
but i would still like to use the new name for this pr
so what we can do is pr the rename to devel, then merge it and pull, i'll do it :trophy: